tests/integration/api_v3/http/blockchainAppsTokenMetadata.test.js

```diff
@@ -29,8 +30,19 @@ const {
 const baseUrl = config.SERVICE_ENDPOINT;
 const baseUrlV3 = `${baseUrl}/api/v3`;
 const endpoint = `${baseUrlV3}/blockchain/apps/meta/tokens`;
+const networkStatusEndpoint = `${baseUrlV3}/network/status`;
 
+let curChainID;
+let curNetwork;
+
+// TODO: Update to use mainnet tokenID/chainID/network when avialble
```

Comment:
Please use `mainnet` values by default to avoid future rework. Kindly reach out to discuss on how to do it locally.

Reply:
Discussed offline.